views/components/table.html

```diff
@@ -0,0 +1,25 @@
+{% macro table(tableData) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{tableData.name}}</h1>
+  </header>
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for header in tableData.data.headers %}
+        <th class="base--th">{{header}}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+        {% for row in tableData.data.row %}
+        <tr class="base--tr" itemprop="dataset" itemscope itemtype="https://schema.org/Dataset">
```

Comment:
This is not a [dataset](https://www.w3.org/wiki/WebSchemas/Datasets). A dataset [is differentiated because](https://en.wikipedia.org/wiki/Data_set) "where every column of the table represents a particular variable, and each row corresponds to a given member of the data set in question"

Not all tables are datasets (and the key thing here is that Dataset is for a whole table, not rows in a table). Datasets you can think of almost as spec sheets. This content, on the other hand, is rather a tabular view of content and actions that can be taken When thinking dataset, consider the [demos provided by WebSchemas](https://logd.tw.rpi.edu/schemaorg_dataset_extension).